requirements/requirements.txt

```diff
@@ -142,7 +142,9 @@ django-crum==0.7.9
 django-extensions==3.2.3
     # via -r /awx_devel/requirements/requirements.in
 django-flags==5.0.13
-    # via -r /awx_devel/requirements/requirements.in
+    # via
+    #   -r /awx_devel/requirements/requirements.in
+    #   django-ansible-base
```

Comment:
this is picking up upstream changes in DAB that happened independently. @zkayyali812 https://github.com/ansible/django-ansible-base/pull/685